foundations/html_css/flexbox/flexbox-intro.md

```diff
@@ -11,6 +11,10 @@ In the following lessons:
 * You will learn about flex-containers and flex-items.
 * You will learn how to create useful components and layouts that go beyond just stacking and centering items.
 
+### Before we get started
+
+Flexbox layouts can get a little complicated. In a previous lesson you learned how to inspect and debug things using your browser's developer tools. Those tools will be _crucial_ for you in the following lessons. Keep it open and keep it handy! If something isn't behaving the way you expect, inspecting it in the developer tools should be you first step _every time_.
```

Comment:
Is it just me but "keep it open and keep it handy" sounds a bit off?

Reply:
lol sounds fine to me... but it is a bit cheeky.  Have an alternative?  Or just leave it out?